app/model/editions/EditionsCollection.scala

```diff
@@ -0,0 +1,62 @@
+package model.editions
+
+import java.time.ZonedDateTime
+import play.api.libs.json.Json
+
+import scalikejdbc.WrappedResultSet
+
+case class EditionsCollection(
+    id: String,
+    name: String,
+    prefill: String,
+    is_hidden: Boolean,
+    metadata: Option[String],
+    updatedOn: Option[ZonedDateTime],
+    updatedBy: Option[String],
+    updatedEmail: Option[String],
+    live: List[EditionsArticle],
```

Comment:
While this makes perfect sense, modelling live and draft at the Issue / Edition level (or the Front level for dotcom fronts) may prove fiddly for the fronts tool 🤔

Reply:
Discussed IRL, this is the  collection model so should be modelling live/draft the same way as normal fronts

Reply:
👍